src/test/ui/issues/issue-76281.rs

```diff
@@ -0,0 +1,7 @@
+// only-wasm32
+// compile-flags: -C opt-level=2 --target wasm32-unknown-unknown
```

Comment:
```suggestion
// compile-flags: -C opt-level=2
```

`only-wasm32` implies that `--target wasm32-unknown-unknown` is used anyway. This test could also be useful on other targets, so `only-wasm32` should also be removed I think.